src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,31 @@
 import React from 'react';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+import { Comments } from '../../types/Comment';
+
+interface CommentInfoProps {
+  comment: Comments
```

Comment:
It's just one Comment
```suggestion
  comment: Comment
```